pkgs/development/python-modules/glyphslib/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fonttools
+, openstep-plist
+, ufoLib2
+, pytestCheckHook
+, unicodedata2
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "glyphslib";
+  version = "6.0.4";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    pname = "glyphsLib";
+    inherit version;
+    sha256 = "PT66n1WEO5FNcwov8GaXT1YNrAi22X4HN7iVNkuehKI=";
+  };
+
+  checkInputs = [ pytestCheckHook ];
+
+  nativeBuildInputs = [ setuptools-scm ];
+
+  propagatedBuildInputs = [
+    fonttools
+    openstep-plist
+    ufoLib2
+    unicodedata2
+  ];
+
+  doCheck = false;
+
+  pythonImportsCheck = [ "glyphsLib" ];
```

```suggestion
  nativeBuildInputs = [ setuptools-scm ];

  propagatedBuildInputs = [
    fonttools
    openstep-plist
    ufoLib2
    unicodedata2
  ];

  checkInputs = [ pytestCheckHook ];

  pythonImportsCheck = [ "glyphsLib" ];
```

If we stick to the standard ordering it makes everything easier and you need to think and search less when debugging.

Also do we really want to disable tests here? If so why? We always want to add a comment.